benchmarks/NonlinearProblem/Bruss_wpd.jmd

```diff
@@ -0,0 +1,124 @@
+---
+title: Ill-Conditioned Nonlinear System Work-Precision Diagrams
+author: Qingyu Qu
+---
+
+# Setups
+
+Fetch required packages
+
+```julia
+using NonlinearSolve, NonlinearSolveMINPACK, SciMLNLSolve, SimpleNonlinearSolve, Sundials
+using LinearAlgebra, SparseArrays, DiffEqDevTools, Plots
+RUS = RadiusUpdateSchemes;
+```
+
+Define the Brussletor problem.
+
+```julia
+const N = 32
```

Comment:
Lets add an axis for increasing the discretization.

Reply:
Increasing N to increase the discretization? Wouldn't that increases the benchmarking time?

Reply:
yes, but we got a big machine